configs/experimental/imagecondition_zero123nerf.yaml

```diff
@@ -131,33 +132,27 @@ system:
       name: None
 
   loss:
-    lambda_sds: 0.1
-    lambda_rgb: 400.0
-    lambda_mask: 50.0
-    lambda_depth: 0.05
-    lambda_depth_rel: [0.0, 0.0, 1.0, 10000]
+    lambda_sds: 10.0
+    lambda_rgb: 0.
+    lambda_mask: 0.
+    lambda_depth: 0.
+    lambda_depth_rel: 0. # [0.0, 0.0, 1.0, 10000]
     lambda_normal: 0. # [0, 0, 0.05, 100]
-    lambda_normal_smooth: 2.0
-    lambda_3d_normal_smooth: 5.0
-    lambda_orient: 0.01
-    lambda_sparsity: 0.01
-    lambda_opaque: 0.05
+    lambda_normal_smooth: 10.0
+    lambda_3d_normal_smooth: 10.0
+    lambda_orient: 1.0
+    lambda_sparsity: 0.1
+    lambda_opaque: 0.1
 
   optimizer:
-    name: Adan
+    name: Adam
```

Comment:
Did you notice any difference specifically tied to Adan vs Adam? Just curious.

Reply:
Noticeably, no, but I didn't do a thorough ablation study as such. I wanted to stick to Adam with a simpler optimizer block, just because I'm more familiar with Adam than Adan.